src/reference/asciidoc/kafka.adoc

```diff
@@ -761,6 +761,28 @@ public KafkaTemplate<String, String> myReplyingTemplate() {
 }
 ----
 
+NOTE: You can annotate a `@KafkaListener` method with `@SendTo` even if no result is returned.
+This is to allow the configuration of an `errorHandler` that can forward information about a failed message delivery to some topic.
+
+[source, java]
+----
+@KafkaListener(id = "voidListenerWithReplyingErrorHandler", topics = "someTopic",
+        errorHandler = "voidSendToErrorHandler")
+@SendTo("failures")
+public void voidListenerWithReplyingErrorHandler(String in) {
```

Comment:
I agree; I just realized while fixing the problem that it was possible; I saw no reason to disable it, though.